### PR TITLE
React capstone project - Digital Currency

### PR DESCRIPTION
**Pull request by RWUBAKWANAYO**
In this project we:

- [ ] Create  React application with  ` react-app`
- [ ] Set up linter
- [ ] Add `routes` to the project
- [ ] Set up  redux
- [ ] Create an action reducer to fetch currency data from API
- [ ] Add reducer to update list of  store's currency 
- [ ] Create an action reducer to fetch currency details from API
- [ ] Add reducer to update details of  store's currency 
- [ ] Test project component, pages, reducers, pure functions and routes
- [ ] Add styles to the project
- [ ] Update readme file
- [ ] Fix linter issues

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    dispatch(FetchCoinsFunc());
+  }, []);
+
+  const handleDetails = (id) => navigate(`/details/${id}`);
+  return (
+    <div data-testid="intro-page" className="data-cont">
+      <Header path="/" />
+      <Chart title1="Crypto" title2="Currency" count={`${coins.length} Coins`} />
+      <div>
+        <h3 className="coins-title">Coins By Name, Symbol, Price</h3>
```
Comment (brytebee) [inline src/pages/Home.js:24]: - [ ] **Required:** You are required to implement a function to filter the data you retrieved from the API and render the result to the page. 

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @RWUBAKWANAYO 👋, 

### Highlights 👍 
- The app is deployed and works great. ✔️ 
- A good video presentation.✔️
- All the tests are well written and passed.✔️
- Well-documented project.✔️
- Awesome job implementing the given design.✔️

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by RWUBAKWANAYO** [comment]
## Resolve changes:

- [ ] In this PR, I use filter method to get specific details from a store instead of making a request

**N.B: Kindly notice that I removed the API request in the details page but I didn't delete the details' action creator and reducer for future use**

**Review by Mre55 — APPROVED** [review]
**Hi @RWUBAKWANAYO,**

# Status: APPROVED 🏆 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 
- The app is deployed and works great ✅ 
- All the tests are well written and passed ✅ 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RWUBAKWANAYO** [comment]
merge changes

**Pull request merged**